editor/components/inserter/test/menu.js

```diff
@@ -90,256 +91,240 @@ const items = [
 	sharedItem,
 ];
 
-describe( 'InserterMenu', () => {
-	// NOTE: Due to https://github.com/airbnb/enzyme/issues/1174, some of the selectors passed through to
-	// wrapper.find have had to be strengthened (and the filterWhere strengthened also), otherwise two
-	// results would be returned even though only one was in the DOM.
+const defaultProps = {
+	position: 'top center',
+	instanceID: 1,
+	items: items,
+	debouncedSpeak: noop,
+	fetchSharedBlocks: noop,
+	setTimeout: noop,
+};
+
+let wrapper;
+
+const setWrapperForProps = ( propOverrides ) => {
+	wrapper = TestUtils.renderIntoDocument(
+		<InserterMenu { ...defaultProps } { ...propOverrides } />
+	);
+};
+
+const initializeMenuDefaultStateAndReturnElement = ( propOverrides ) => {
+	setWrapperForProps( propOverrides );
+	/* eslint-disable react/no-find-dom-node */
```

Comment:
We should consider overriding test files in ESLint to allow for `findDOMNode`

https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files

Reply:
> We should consider overriding test files in ESLint to allow for findDOMNode

That would be good, should be done in a separate pull though?